package.json

```diff
@@ -59,6 +59,7 @@
   },
   "dependencies": {
     "classnames": "^2.2.5",
+    "core-js": "^2.5.7",
```

Comment:
but aren't these dev dependencies? 🤔

Reply:
No... they would be runtime. That is the main idea i guess - would like this to run on the physical machine and validate responses on the fly...